infrastructure/modules/dynamodb/variables.tf

```diff
@@ -24,3 +24,7 @@ variable "attributes" {
     type = "S"
   }]
 }
+
+variable "deletion_protection_enabled" {
+  default = false
```

Comment:
Approved but wondering why would the default be false when we setting it to true in the stacks - what is the reasoning for it to be false?

Reply:
In this repo it probably doesn't matter to be fair as for now I think we only create the one dynamodb table. But in the service-management repo we create multiple tables per terraform workspace (to hold test data)  and we want to be able to tear these down without issue when we clear out the workspace. So I opted to not protect every table by default but protect the one we care about most - the terraform lock table.